### ISSUE TITLE
0.15 release broke 0.14

### ISSUE THREAD

**Issue by antonb-via** [Koka/odbc-rs#126]
Hi,
please be aware that 0.15 release broke 0.14 version:

```

   Compiling odbc v0.14.0
error[E0107]: wrong number of type arguments: expected 1, found 0
 --> /home/vsts/.cargo/registry/src/github.com-1ecc6299db9ec823/odbc-0.14.0/src/connection.rs:7:11
  |
7 |     safe: safe::Connection<'env>,
  |           ^^^^^^^^^^^^^^^^^^^^^^ expected 1 type argument
```

**Comment by Koka** [comment]
Sorry, that's 0.14.1 - will yank it

**Comment by Koka** [comment]
Hmm actually 0.14.1 was not released on crates.io

**Comment by Koka** [comment]
@antonb-via Should be fixed in 0.14.2
